packages/cardhost/app/templates/components/card-creator.hbs

```diff
@@ -46,21 +46,15 @@
   </section>
 </CardhostLeftEdge>
 
-<aside class="cardhost-right-edge" data-test-cardhost-right-edge>
-  <section class="cardhost-right-edge-panel">
-    <h3 class="cardhost-right-edge-panel--header">
-      Edit Field Schema
-    </h3>
-    {{#if this.selectedField}}
-      <FieldRenderer
-        @field={{this.selectedField}}
-        @mode="schema"
-        @setNeededWhenEmbedded={{action this.setNeededWhenEmbedded}}
-        @setFieldName={{action this.setFieldName}}
-      />
-    {{/if}}
-  </section>
-</aside>
+<RightEdge
+  @selectedField={{this.selectedField}}
+  @selectedFieldName={{this.selectedFieldName}}
+  @selectedFieldLabel={{this.selectedFieldLabel}}
```

Comment:
I'm confused by this: `selectedFieldName` and `selectedFieldLabel` being passed in. These are already properties of the `@selectedField`, which already has tracked properties of `@selectedField.name` and `@selectedField.label`. Why are we duplicating this instead of pulling this from the @selectedField arg?